### PR TITLE
Extend critical health checks

### PR DESCRIPTION
**Pull request by kwypchlo**
- added "website" check that makes sure main page responds ok and tests `siasky.net` ssl certificate
- added "skylink_via_subdomain" check that makes sure that people can access a skylink via subdomain and tests `*.siasky.net` ssl certificate
- added "hns_via_subdomain" check that makes sure that handshake domains work and tests `*.hns.siasky.net` ssl certificate
- added "account_website" check that makes sure that account dashboard website is accessible

### CODE REVIEW

File: packages/health-check/src/checks/critical.js [hunk 1]
```diff
@@ -25,24 +28,45 @@ async function uploadCheck(done) {
...
+}
+
+// skylinkSubdomainCheck returns the result of downloading the hard coded link via subdomain
+async function skylinkSubdomainCheck(done) {
+  const url = await skynetClient.getSkylinkUrl(exampleSkylink, { subdomain: true });
+
+  return genericAccessCheck("skylink_via_subdomain", url, done);
+}
+
+// skylinkSubdomainCheck returns the result of downloading the hard coded link via subdomain
```
Comment (MSevey) [inline packages/health-check/src/checks/critical.js:53]: Follow Up, copy paste docstring.

File: packages/health-check/src/checks/critical.js [hunk 2]
```diff
@@ -25,24 +28,45 @@ async function uploadCheck(done) {
...
+async function handshakeSubdomainCheck(done) {
+  const url = await skynetClient.getHnsUrl("note-to-self", { subdomain: true });
+
+  return genericAccessCheck("hns_via_subdomain", url, done);
+}
+
+async function accountHealthCheck(done) {
```
Comment (MSevey) [inline packages/health-check/src/checks/critical.js:60]: Follow Up: docstrings for other functions.

**Review by MSevey — APPROVED** [review]

File: packages/health-check/src/checks/critical.js (hunk 2 above)
Comment (kwypchlo) [reply packages/health-check/src/checks/critical.js:66]: added

File: packages/health-check/src/checks/critical.js (hunk 1 above)
Comment (kwypchlo) [reply packages/health-check/src/checks/critical.js]: changed

File: packages/health-check/Dockerfile [hunk 3]
```diff
@@ -2,14 +2,14 @@ FROM node:16.1.0-alpine
 
 WORKDIR /usr/app
 
+RUN echo '*/5 * * * * /usr/app/cli/run critical > /dev/stdout' >> /etc/crontabs/root
```
Comment (MSevey) [inline packages/health-check/Dockerfile:5]: Follow Up: Should add some comments to this dockerfile explaining the steps and why the order is important.

File: packages/health-check/src/checks/critical.js [hunk 4]
```diff
@@ -25,24 +28,53 @@ async function uploadCheck(done) {
...
+}
+
+// handshakeSubdomainCheck returns the result of downloading the skylink via handshake domain
+async function handshakeSubdomainCheck(done) {
+  const url = await skynetClient.getHnsUrl("note-to-self", { subdomain: true });
+
+  return genericAccessCheck("hns_via_subdomain", url, done);
+}
+
+// handshakeSubdomainCheck returns the result of accessing account dashboard website
```
Comment (MSevey) [inline packages/health-check/src/checks/critical.js:60]: Copy paste :-)

**Review by MSevey — APPROVED** [review]

**Review by m-cat — DISMISSED** [review]

**Review by m-cat — APPROVED** [review]

**Review by MSevey — APPROVED** [review]

**Pull request merged**